discord-stubs/ext/commands/core.pyi

```diff
@@ -37,7 +37,8 @@ class _CheckPredicate(Protocol[_T_contra]):
     def __call__(self, __ctx: _T_contra) -> Union[bool, Coroutine[Any, Any, bool]]: ...
 
 @type_check_only
-class _CheckDecorator(Protocol):
+class _CheckDecorator:
+    predicate: _CheckPredicate[_CT]
```

Comment:
To keep `_CheckDecorator` so it's not generic, `_CT` needs to be chaned to `Any` here

Reply:
Would it be reasonable to set `discord.ext.commands.Context`, since that will always be a Context instance?